src/components/Home/SubmissionFolderList.tsx

```diff
@@ -144,16 +144,18 @@ const SubmissionFolderList: React.FC<any> = () => {
   // Full list of folders
   const Submissions = () => (
     <Grid item xs={12} className={classes.tableCard}>
-      <SubmissionIndexCard
-        folderType={location === "drafts" ? FolderSubmissionStatus.unpublished : FolderSubmissionStatus.published}
-        folders={location === "drafts" ? unpublishedFolders : publishedFolders}
-        location={location}
-        page={page}
-        itemsPerPage={itemsPerPage}
-        totalItems={location === "drafts" ? totalFolders.totalUnpublishedFolders : totalFolders.totalPublishedFolders}
-        fetchItemsPerPage={handleFetchItemsPerPage}
-        fetchPageOnChange={handleFetchPageOnChange}
-      />
+      {
```

Comment:
Is there an extra `{`  ?

Reply:
Good catch, this seems to be unnecessary.

Reply:
Change is pushed to the branch.